### PR TITLE
If not authorized on an audio file, show the default waveform poster

### LINKED ISSUE #1761: image indicating audio should display prior to user login
### Expected behavior
There is an image used that indicates something is an audio media object. This image should display prior to the user logging in.

### Actual behavior
The current screen is blank/white and shows a scrub bar at the bottom.
Screenshot 2023-11-16 at 3 57 39 PM

### Steps to reproduce the behavior
If already authenticated in, open in incognito window: https://sul-purl-stage.stanford.edu/yh788wh7909

### PR DESCRIPTION
**Pull request by aaron-collier**
Closes #1761 

When not authorized and the file is audio:

Screenshot 2023-12-04 at 1 57 31 PM

Audio when a poster is provided:

Screenshot 2023-12-05 at 4 20 33 PM

Video shows the poster:

Screenshot 2023-12-04 at 1 59 13 PM

### CODE REVIEW

**Review by jcoyne — CHANGES_REQUESTED** [review]
Shouldn't this support the provided poster image if one is provided when logged out? For example, I think https://sul-purl-stage.stanford.edu/hj997tt0155 should display the tape.

**Comment by aaron-collier** [comment]
@jcoyne suggestion on how to select that image as the poster? The `poster` attribute on the audio player is the waveform image:

```
<audio id="sul-embed-media-0" data-auth-url="https://sul-stacks-stage.stanford.edu/media/hj997tt0155/bh009rf1857_a_sl.m4a/auth_check" data-media-tag-target="mediaTag" poster="https://embed-stage.stanford.edu/assets/waveform-audio-poster-2120200a7da69d5cb6a3937536d602e0aeaa6086.svg" controls="controls" crossorigin="anonymous" class="sul-embed-media-file" height="100%"><source src="https://sul-streaming-stage.stanford.edu/stacks/_definst_/hj/997/tt/0155/mp4:bh009rf1857_a_sl.m4a/playlist.m3u8" type="application/x-mpegURL"></audio>
```

**Comment by jcoyne** [comment]
@aaron-collier you're right my mistake.  We should find/create a test case that actually has a thumbnail though.

**Comment by andrewjbtw** [comment]
Prod example with a dedicated poster for the item: https://purl.stanford.edu/jz314xw3742

**Comment by andrewjbtw** [comment]
I can make that into a stage example with restricted rights.

**Comment by andrewjbtw** [comment]
I posted on Slack too, but here's an audio poster with Stanford restrictions in stage: https://sul-purl-stage.stanford.edu/tw581qt4975

File: app/components/embed/media_with_companion_windows_component.html.erb [hunk 1]
```diff
@@ -112,6 +112,13 @@
         </picture>
       </div>
 
+      <div data-controller="locked-audio-poster" data-action="auth-success@window->locked-audio-poster#hide auth-denied@window->locked-audio-poster#show" tabindex="-1" role="presentation" style="display: none; flex: 1 0 100%;">
```
[the remark this answers is not in the document]
Comment (aaron-collier) [reply app/components/embed/media_with_companion_windows_component.html.erb:115]: Sure. Using the display style is what is done for the media poster, so I was following that pattern. Is that something that should be updated on both?

**Comment by aaron-collier** [comment]
@jcoyne regarding behavior: the `filter` here would select whichever one has a iiif `default.jpg` (this is probably a bit fragile so there may be a better way to determine if an object isn't using the default poster).

**Comment by jcoyne** [comment]
@aaron-collier do we need to care?  Can't we just grab the poster attribute and put it in the image tag?

**Comment by aaron-collier** [comment]
Ah, are you suggesting to just use the poster value whether it's default or not? Yeah, I can certainly do that.

**Review by jcoyne — CHANGES_REQUESTED** [review]

**Review by jcoyne — APPROVED** [review]

**Pull request merged**